Dockerfile

```diff
@@ -5,6 +5,12 @@ ARG PROFILE=release
 WORKDIR /cerenetwork
 COPY . /cerenetwork
 
+# Copy smart-contract artifacts (cere01.wasm and metadata.json)
+ARG SMART_CONTRACT_VERSION=1.1.0
+FROM cere-enterprise-smart-contracts/smart-contracts:$SMART_CONTRACT_VERSION as smart-contracts
```

Comment:
Are you using multistage build or you need this image just to copy files?
If the second one, so --from could be specified with image name.
Also, image name should be specified fully, with ECR repo.